support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -327,12 +329,33 @@ EventsAndTriggers:
           # for visualization.
           CoordinatesFloatingPointType: Float
           FloatingPointTypes: [Float]
-  # Try to find common horizon at small separation
+# Trigger to find common horizon and gather volume data needed for ringdown.
   - Trigger:
-      SeparationLessThan:
-        Value: 2.5
+      And:
+        - SeparationLessThan:
+            Value: 2.38 # for q=1, spin=0, quasicircular
+        - Slabs:
+            EvenlySpaced:
+              Interval: 1
+              Offset: 0
     Events:
+      - ChangeSlabSize:
+          DelayChange: 0
+          StepChoosers:
+            - Constant: 0.01
       - ObservationAhC
+      - ObserveFields:
+          SubfileName: ForContinuation
+          VariablesToObserve:
+            - Lapse
+            - ConstraintEnergy
```

Comment:
I'm a bit surprised we write out the constraint energy and lapse. I think these could be computed from the 3 evolved variables. It would be good to have a comment why they are observed instead of computed when needed.

Reply:
Oh, we don't think we need to output those for the transition to ringdown script to work, they were originally there to see where the constraints were blowing up or getting large. I can take that out

Reply:
For the pipeline and transition to ringdown, yeah we probably don't need these.